frontend/src/components/SatelliteTable.vue

```diff
@@ -198,6 +195,7 @@ export default {
       user: this.$store.state.loggedUser,
       messageJoinCanteen: null,
       unlinkConfirmationOpen: false,
+      unlinkItem: undefined,
```

Comment:
En général pour ces cas je préfère explicitement mettre `null` pour laisser `undefined` pour les variables non définies ou non attendues (comme ça c'est plus cohérent avec le reste de fichier aussi)

```suggestion
      unlinkItem: null,
```